src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,40 @@
 'use strict';
 
+const DEFAULT_HEALTH = 100;
+const DAMAGE_VALUE = 50;
+
 class Animal {
-  // write your code here
+  static alive = [];
+  constructor(name, health = DEFAULT_HEALTH) {
+    this.health = health;
+    this.name = name;
+    Animal.alive.push(this);
+  }
+
+  removeDead() {
+    Animal.alive = Animal.alive.filter((item) => item.health > 0);
+  }
 }
 
 class Herbivore extends Animal {
-  // write your code here
+  constructor(health, name) {
+    super(health, name);
+    this.hidden = false;
+  }
+
+  hide() {
```

Comment:
[FUNCTION NAMING] The `hide` method should only set the `hidden` property to `true` as per the task description. If there is a requirement to unhide the animal, a separate method should be implemented.